specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/definitions.json

```diff
@@ -8761,6 +8761,73 @@
         "productId"
       ],
       "description": "Tag-product link entity properties."
+    },
+    "GatewayListDebugCredentialsContract": {
+      "type": "object",
+      "properties": {
+        "credentialsExpireAfter": {
+          "type": "string",
+          "format": "duration",
+          "description": "After what period of time to make credentials expire."
```

Comment:
>expire [](http://example.com/codeflow?start=72&length=6)

suggestion: this documentation is even more helpful with examples showing the duration format

Reply:
Updated description.